docs/man/borg.1

```diff
@@ -460,6 +460,10 @@ When set to a numeric value, this determines the maximum \(dqtime to live\(dq fo
 entries (default: 20). The files cache is used to determine quickly whether a file is unchanged.
 The FAQ explains this more detailed in: \fIalways_chunking\fP
 .TP
+.B BORG_USE_CHUNKS_ARCHIVE
+When set to no (default: yes), the chunks.archive.d folder will not be used. This reduces
+disk space usage but slows down cache resyncs.
+.TP
```

Comment:
You don't need to update that file, it will be generated from the docs when I do a release.